README.md

```diff
@@ -1,5 +1,18 @@
 # trek
 
+Trek is a tool that assists in postgres schema development and deployment. Trek provides a unified and simple workflow for developers to change the database schema and commit versioned migration files to a SCM such as git. Trek also helps with rolling out those migrations to a live database.
+
+A basic trek workflow looks like this:
+- Initialize a trek directory `mkdir my_database; cd my_database; trek init`
+- Change the database schema by using the [pgModeler](https://github.com/pgmodeler/pgmodeler) GUI.
+- Run trek to generate a new migration file. The migration file will contain all changes since the last version as postgres DDL (ALTER TABLE, etc.). The migration file is stored as file in the `my_database/migrations` folder.
```

Comment:
```suggestion
- Run trek to generate a new migration file. The migration file will contain all changes since the last version as PostgreSQL DDL (ALTER TABLE, etc.). The migration file is stored as file in the `my_database/migrations` folder.
```